test/common/deployPipelineCache.test.ts

```diff
@@ -63,10 +64,6 @@ describe('deployPipelineCache', () => {
     expect(excThrown);
   });
 
```

Comment:
This test was creating a similar function so just reused the same code from utils